Task2/Task2/Task2/Products/Screw.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Task2.Exceptions;
+
+namespace Task2.Products
+{
+    public class Screw : Product
+    {
+        public Screw()
+        {
+
+        }
+
+        public Screw(
+            string productName, 
+            double purchaseCost, 
+            double margin, 
+            int numberOfUnits)
+            : base(
+                productName, 
+                purchaseCost, 
+                margin, 
+                numberOfUnits)
+        {
+
+        }
+
+        public static Screw operator +(Screw leftProduct, Screw rightProduct)
```

Comment:
The same issue about arguments null check.

Reply:
fixed